### PR TITLE
Remove scratchwork appendix

### PR DESCRIPTION
**Pull request by domenic**
It detracts from the message

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,85 +71,5 @@ Signed,
 - Sindre Sorhus ([@sindresorhus](https://github.com/sindresorhus)), maintainer of Yeoman
 - Trek Glowacki ([@trek](https://github.com/trek)), a maintainer of Ember.js
 - Sashko Stubailo ([@stubailo](https://github.com/trek)), a maintainer of Meteor
-- Graeme Yeates ([@megawac](https://github.com/megawac)), a maintainer of Backbone and Underscore
```
Comment (thejameskyle) [inline README.md]: Can you update your commit to not remove Graeme?

Comment (domenic) [inline README.md]: Race condition lol

**Comment by domenic** [comment]
Fixed

**Comment by sindresorhus** [comment]
It's still useful though. I would just move it into another file.

**Comment by domenic** [comment]
It's always in Git history.

**Pull request merged**